x-pack/plugins/monitoring/public/components/logstash/listing/listing.js

```diff
@@ -72,7 +72,7 @@ class ListingUI extends PureComponent {
         name: i18n.translate('xpack.monitoring.logstash.nodes.eventsIngestedTitle', {
           defaultMessage: 'Events Ingested'
         }),
-        field: 'events_out',
+        field: 'events_ingested',
```

Comment:
Should we just rename https://github.com/elastic/kibana/pull/35122/files#diff-41737f3c90b33f58268178cc5aa38d20L122 to be `events_out`?

Reply:
I'm good with that, unless you had some specific reason to call it `events_ingested` in the first place? https://github.com/elastic/kibana/commit/9c842eecca768b056a0431f198450cdf1abb6603#diff-41737f3c90b33f58268178cc5aa38d20R121

Reply:
No good reason. Let's go with `events_out` in both places

Reply:
Made the change in 670ea02. Ready for re-review.